OPS-61 DN Initial

jamstu26 opened the pull request.

Commit:
Create nzdn.md

motionaviation committed:
Create nzdn-airspace.png

Commit:
Airspace and initial NZTI notes

motionaviation committed:
Images

motionaviation committed:
Added Areas of Resp

Commit:
Initial SID assignment table

Commit:
changes to sid table, pushback and taxi notes

Commit:
That table was really annoying me

Commit:
visual dep, more info on TI and taxi instruction

Commit:
thanks mr. sumner

Commit:
Update nzdn.md

remove clearance delivery, move sid table under departures, started arrivals section

Commit:
Update nzdn.md

motionaviation committed:
Minor Tweaks

Commit:
Update nzdn.md

rb-vatnz commented on docs/aerodromes/Class-D/nzdn.md:
```diff
@@ -0,0 +1,116 @@
+
+# Pushback and Taxi
+
+Pushback and startup clearence from stand one to five is managed by TDN. Controllers can make use of a "tail-facing" instruction to manage direction of pushback. Aircraft on taxiway `C` must contact TDN at `C2` holding point before proceeding.
```
```suggestion
Pushback and startup clearance from stand one to five is managed by Tower. Controllers can make use of a "tail-facing" instruction to manage direction of pushback. Aircraft on taxiway `C` must contact Tower at `C2` holding point before proceeding.
```

rb-vatnz commented on docs/aerodromes/Class-D/nzdn.md:
```diff
@@ -0,0 +1,116 @@
+
+Pushback and startup clearence from stand one to five is managed by TDN. Controllers can make use of a "tail-facing" instruction to manage direction of pushback. Aircraft on taxiway `C` must contact TDN at `C2` holding point before proceeding.
+
+An aircraft may be taxiied to the holding point, traffic permitting the aircraft may be given backtrack as apart of the taxi clearance and also traffic permitting the controller may choose to omit the taxiway. Aircraft may advise the controller they are able to accept an intersection departure, therefore they may be taxiied to the holding point, omiting the backtrack instruction.
```
```suggestion
An aircraft may be taxiied to the holding point and, traffic permitting the aircraft may be given backtrack as apart of the taxi clearance. Traffic permitting, the controller may also choose to omit the taxiway. Aircraft may advise the controller they are able to accept an intersection departure, therefore they may be taxiied to the holding point, omiting the backtrack instruction.
```

rb-vatnz commented on docs/aerodromes/Class-D/nzdn.md:
```diff
@@ -0,0 +1,116 @@
+| 03     | `SW #`     | Cat A to D             | To `BIDEL`, `MEGUG` `MIPAK`, `PORUT` and `QN`. Non-RNAV.                                                                                  |
+
+## Visual Departures
+When either runway is in use, TDN may issue northern-bound aircraft a visual departure direct to `BIDEL` on pilot's request, traffic permitting. Alternatively, when Runway 21 is in use, TDN may issue northern-bound aircraft on the `BIDEL #Q`or `BIDEL #S` a visual departure to `ATGUB` or `IDLIK` on pilot's request ([Refer to AIP chart for `BIDEL #Q` and `BIDEL #S`](https://www.aip.net.nz/assets/AI ...
```
```suggestion
When either runway is in use, Tower may issue northern-bound aircraft a visual departure direct to `BIDEL` on pilot's request, traffic permitting. Alternatively, when Runway 21 is in use, Tower may issue northern-bound aircraft on the `BIDEL #Q`or `BIDEL #S` a visual departure to `ATGUB` or `IDLIK` on pilot's request ([Refer to AIP chart for `BIDEL #Q` and `BIDEL #S`](https://www.aip.net.nz/assets/AIP/Aerodrome-Charts/Dunedin-NZDN/NZDN_62.7_62.8.pdf)).
```

rb-vatnz commented on docs/aerodromes/Class-D/nzdn.md:
```diff
@@ -0,0 +1,116 @@
+All arrivals terminate at `AKLAR`, which is the IAF for the RNAV and ILS approach.
+
+### Missed Approach
+In the event of a missed approach, aicraft should be instructed to carry out the published missed approach procedure. For the ILS and NDB procedures, this will take the aircraft straight out to `MI` NDB, then a right turn to `BE` NDB holding at `A040`. For the RNAV procedure, it will take the aircraft to `DABCA` holding at `A040`. The aircraft can then be issued the approprite arrival procedure f ...
```
```suggestion
In the event of a missed approach, aircraft should be instructed to carry out the published missed approach procedure. For the ILS and NDB procedures, this will take the aircraft straight out to `MI` NDB, then a right turn to `BE` NDB holding at `A040`. For the RNAV procedure, it will take the aircraft to `DABCA` holding at `A040`. The aircraft can then be issued the appropriate arrival procedure from `DABCA` to commence the approach again.
```

rb-vatnz commented on docs/aerodromes/Class-D/nzdn.md:
```diff
@@ -0,0 +1,116 @@
+
+## NZTI - Taieri
+
+Taieri is a small, uncontrolled GA aerodrome located 8nm Northeast of NZDN (Dunedin). Aircraft may operate within the TI circuit and within 3nm of the aerodrome below `A011` without reference to ATC. Aircraft departing TI shall recieve clearence and departure instructions from TDN prior to departure.
```
```suggestion
Taieri is a small, uncontrolled GA aerodrome located 8nm Northeast of NZDN (Dunedin). Aircraft may operate within the TI circuit and within 3nm of the aerodrome below `A011` without reference to ATC. Aircraft departing TI shall receive clearance and departure instructions from TDN prior to departure.
```

mwr2410 committed:
Update docs/aerodromes/Class-D/nzdn.md

Co-authored-by: rb-vatnz <125234863+rb-vatnz@users.noreply.github.com>

mwr2410 committed:
Update docs/aerodromes/Class-D/nzdn.md

Co-authored-by: rb-vatnz <125234863+rb-vatnz@users.noreply.github.com>

mwr2410 committed:
Update docs/aerodromes/Class-D/nzdn.md

Co-authored-by: rb-vatnz <125234863+rb-vatnz@users.noreply.github.com>

mwr2410 committed:
Update docs/aerodromes/Class-D/nzdn.md

Co-authored-by: rb-vatnz <125234863+rb-vatnz@users.noreply.github.com>

mwr2410 committed:
Update docs/aerodromes/Class-D/nzdn.md

Co-authored-by: rb-vatnz <125234863+rb-vatnz@users.noreply.github.com>

mwr2410 committed:
review of NZDN

- Content ok
- Fixed most headings, which were wrong format
- Applied Richard's suggestions as per github
- Needs to be finished beyond coordination section

mwr2410 reviewed (changes requested):
Finish co-ordination and onwards - refer to Connor

motionaviation committed:
Add Coord

dirtyformal commented:
Build is failing due to not having the updated MkDocs file. Will require an edit from me before merged.

dirtyformal committed:
Merge branch 'main' into pr/43

dirtyformal committed:
Review. Add NV TWR Coord

mwr2410 committed:
add procedural tower index

added new section - procedural towers
relocated page to new section
